Modules/faulthandler.c

```diff
@@ -1327,26 +1395,26 @@ PyExec_faulthandler(PyObject *module) {
     /* Add constants for unit tests */
 #ifdef MS_WINDOWS
     /* RaiseException() codes (prefixed by an underscore) */
-    if (PyModule_AddIntConstant(module, "_EXCEPTION_ACCESS_VIOLATION",
-                                EXCEPTION_ACCESS_VIOLATION)) {
+    if (PyModule_Add(module, "_EXCEPTION_ACCESS_VIOLATION",
+                     PyLong_FromUnsignedLong(EXCEPTION_ACCESS_VIOLATION))) {
```

Comment:
Should we add a test that `faulthandler._EXCEPTION_ACCESS_VIOLATION == 0xc0000005`, or at least that it is not negative?